app/src/main/java/com/asfoundation/wallet/ui/TransactionsActivity.java

```diff
@@ -154,7 +159,8 @@ public static Intent newIntent(Context context) {
 
   private void onFetchTransactionsError(Double maxBonus) {
     if (emptyView == null) {
-      emptyView = new EmptyTransactionsView(this, this, String.valueOf(maxBonus));
+      emptyView =
+          new EmptyTransactionsView(this, String.valueOf(maxBonus), emptyTransactionsSubject, this);
```

Comment:
I’m not sure, but I think that if you move the systemView.showEmpty outside of the if, it should solve the problem of launching more than 1 intent